charts/airflow/templates/_helpers/pods.tpl

```diff
@@ -60,6 +60,8 @@ EXAMPLE USAGE: {{ include "airflow.init_container.check_db" (dict "Release" .Rel
 {{- define "airflow.init_container.check_db" }}
 - name: check-db
   {{- include "airflow.image" . | indent 2 }}
+  resources:
+    {{- toYaml .resources | nindent 4 }}
```

Comment:
The value `.resources` will not be defined, you must also change every time the `"airflow.init_container.check_db"` template is called to include the appropriate resources.

For example, in the scheduler this might look like:

```yaml
# top of file
{{- $resources := .Values.scheduler.resources }}

# in the init container section
{{- include "airflow.init_container.check_db" (dict "Release" .Release "Values" .Values "volumeMounts" $volumeMounts "resources" $resources) | indent 8 }}
```